system/Commands/ListCommands.php

```diff
@@ -115,82 +107,43 @@ public function run(array $params)
 	{
 		$commands = $this->commands->getCommands();
 
-		$this->describeCommands($commands);
-
-		CLI::newLine();
-	}
-
-	//--------------------------------------------------------------------
-
-	/**
-	 * Displays the commands on the CLI.
-	 *
-	 * @param array $commands
-	 */
-	protected function describeCommands(array $commands = [])
-	{
 		ksort($commands);
 
 		// Sort into buckets by group
-		$sorted         = [];
-		$maxTitleLength = 0;
+		$groups = [];
 
 		foreach ($commands as $title => $command)
 		{
-			if (! isset($sorted[$command['group']]))
+			if (! isset($groups[$command['group']]))
 			{
-				$sorted[$command['group']] = [];
+				$groups[$command['group']] = [];
 			}
 
-			$sorted[$command['group']][$title] = $command;
-
-			$maxTitleLength = max($maxTitleLength, strlen($title));
+			$groups[$command['group']][$title] = $command;
 		}
 
-		ksort($sorted);
+		$length = max(array_map('strlen', array_keys($commands)));
+
+		ksort($groups);
 
 		// Display it all...
-		foreach ($sorted as $group => $items)
+		foreach ($groups as $group => $commands)
 		{
-			CLI::newLine();
-			CLI::write($group);
-
-			foreach ($items as $title => $item)
+			CLI::write($group, 'yellow');
+			foreach ($commands as $command)
 			{
-				$title = $this->padTitle($title, $maxTitleLength, 2, 2);
-
-				$out = CLI::color($title, 'yellow');
-
-				if (isset($item['description']))
-				{
-					$out .= CLI::wrap($item['description'], 125, strlen($title));
-				}
+				CLI::write(
+					CLI::color($this->setPad($command['name'], $length, 2, 2), 'green')
+					 . $command['description']
```

Comment:
If we'll remove the wrap here then long descriptions would span across the terminal width and overflow.